test/tagbot-unit.jl

```diff
@@ -36,6 +36,8 @@ end
     @test TB.repo_and_version_of_pull_request_body(ssh) == ("Foo/Bar", "v1.2.3")
     gitlab = body("https://gitlab.com/Foo/Bar")
     @test TB.repo_and_version_of_pull_request_body(gitlab) == (nothing, "v1.2.3")
+    nobody = nothing
```

Comment:
```suggestion
no_body = nothing
```

Reply:
Fixed in 3fbe6e210c591c01596de92d6911f7a86c9eda97.